src/main/webapp/filesFragment.xhtml

```diff
@@ -35,15 +35,16 @@
     <p:remoteCommand name="rebindCommand" process="@this" update="filesTable" oncomplete="javascript:bind_bsui_components();"/>
 
     <!-- Files Table -->
+    <!-- TABLESTYLE + WIDTH:AUTO FIX COL WIDTH OVERFLOW HIDDEN, MIN-WIDTH 100% ADDED FOR TABLE CONTENT LESS THAN 100% https://stackoverflow.com/a/38582318 -->
     <p:dataTable id="filesTable" 
                  rows="10" paginator="#{DatasetPage.fileMetadatasSearch.size() gt 10}" paginatorPosition="bottom"
                  paginatorTemplate="{FirstPageLink} {PreviousPageLink} {PageLinks} {NextPageLink} {LastPageLink} #{bundle['file.dynamicCounter.filesPerPage']} {RowsPerPageDropdown}"
                  rowsPerPageTemplate="10,25,50"
-                 style="margin-right:1px;"
+                 style="margin-right:1px;" tableStyle="min-width:100%;width:auto;"
                  value="#{DatasetPage.fileMetadatasSearch}"
                  rowIndexVar="rowNum" rowKey="#{fileMetadata.dataFile.storageIdentifier}"
                  selection="#{DatasetPage.selectedFiles}" var="fileMetadata" widgetVar="filesTable"
-                 rendered="#{DatasetPage.fileDisplayTable and empty DatasetPage.editMode and (DatasetPage.workingVersion != null)}"
```

Comment:
is this removed because the filefragment itself is only shown when not in editMode?

Reply:
May have been a lil too aggressive with this render logic clean up. I believe that fileDisplayTable is required to toggle the view to support file hierarchy. I will review this.

Wanted to remove editMode logic as the filesFragment is only included on the dataset pg. There is also render logic on the tab containing the include for editMode, and that too may be kosher to remove.

Reply:
@mheppler right, but dataset pg does have the different edit modes; so we can't generally remove editMode. Specifically, though, if this is a case where this fragment is *only* included within an element that alread checks edit mode then we're probably ok removing here.

Reply:
OK, found the code where it is included - do note it does have the editmode checked in the rendered, so from a logic sense, it would be ok to remove. If we ever decide to use this fragment for something else, we'd also need to make sure this was the case, or re add here. 

```                            <p:tab id="dataFilesTab" title="#{bundle.files}" 
                                   rendered="#{(!DatasetPage.workingVersion.deaccessioned 
                                               or (DatasetPage.workingVersion.deaccessioned and DatasetPage.canUpdateDataset())) 
                                               and (empty DatasetPage.editMode)}">
                                <ui:include src="filesFragment.xhtml">
                                    <ui:param name="fileDownloadHelper" value="#{DatasetPage.fileDownloadHelper}"/>
                                </ui:include>
                            </p:tab>
```

One thing I started to do is at a comment at the top of new fragments explaining their use a little (see the kebab related fragments). Can you add some comments to the top of this (basically saying that it is currently only used by dataset page in a case where editmode is empty)?

Reply:
we added some comments to the fragment that is is only included with empty editMode. (so if we reuse this some day elsewhere, we know to check that)